torchtune/rlhf/loss/dpo.py

```diff
@@ -160,87 +160,6 @@ def forward(
         return losses, chosen_rewards, rejected_rewards
 
 
-class IPOLoss(nn.Module):
-    r"""
-    Identity Preference Optimization (IPO) Loss module: https://arxiv.org/abs/2310.12036.
-    Intuition from the paper:
-
-        (Given a policy :math:`\pi` and reference policy, :math:`\pi_{\text{ref}}`)
-
-        IPO learns from preferences dataset simply by regressing the gap between log-likelihood ratios
-
-        :math:`\log \bigg(\frac{(\pi(\text{chosen})}{\pi(\text{rejected})}\bigg)` and :math:`\log \bigg(\frac{\pi_{\text{ref}}(\text{chosen})}{\pi_{\text{ref}}(\text{rejected})} \bigg)`
```

Comment:
my beautiful latex : (

Reply:
Its not a goodbye. Its just a "until next time".

Thanks for all the reviews and help with this @SalmanMohammadi !